### PR TITLE
Release/web/redshift/1.1.0

### PR DESCRIPTION
**Pull request by colmsnowplow**

### CODE REVIEW

File: .test/great_expectations/config/config_variables.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# Substitution values defined here can be a simple (non-nested) value,
+# nested value such as a dictionary, or an environment variable (i.e. ${ENV_VAR})
+#
+#
+# https://docs.greatexpectations.io/en/latest/how_to_guides/configuring_data_contexts/how_to_use_a_yaml_file_or_environment_variables_to_populate_credentials.html
+
+instance_id: 9a5bf883-42c2-49c3-9128-a707efea14d3
+redshift:
+  drivername: postgresql+psycopg2
+  host: sp-com-snplow-eng-aws-prod1-redshift-cluster.cdzqkzl74pqa.eu-central-1.redshift.amazonaws.com
```
Comment (paulboocock) [inline .test/great_expectations/config/config_variables.yml]: Remove hardcoded URL

File: README.md [hunk 2]
```diff
@@ -1 +1,36 @@
-Snowplow Data Models Repo
+![snowplow-logo](snowplow_logo.jpg)
+
+Snowplow is a scalable open-source platform for rich, high quality, low-latency data collection. It is designed to collect high quality, complete behavioural data for enterprise business.
+
+# Snowplow Pipeline Overview
+
+![snowplow-pipeline](snowplow_pipeline.png)
```
Comment (paulboocock) [inline README.md]: Can we fix this image to not have the autocorrect squiggles?

File: README.md [hunk 3]
```diff
@@ -1 +1,36 @@
...
+
+The [Snowplow trackers](https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/) enable highly customisable collection of raw, unopinionated event data. The pipeline validates these events against a JSONSchema - to guarantee a high quality dataset - and adds information via both standard and custom enrichments.
+
+This data is then made available in-stream for real-time processing, and can also be loaded to blob storage and data warehouse for analysis.
+
+The SQL data models in this repo aggregate those raw events in-warehouse, to produce consumable datasets for analysis and actionable insight to be built upon. The models are built to allow the addition of custom use cases and aggregations, in a robust, scalable way.
+
+# Prerequisites
+
+These models are written in a format that is runnable via [SQL-runner](https://github.com/snowplow/sql-runner) - available for download from [Bintray](https://dl.bintray.com/snowplow/snowplow-generic/).
```
Comment (paulboocock) [inline README.md]: Add a more direct link to sql-runner on bintray?

**Review by paulboocock — APPROVED** [review]
LGTM

**Comment by paulboocock** [comment]
Should we also link to other documentation from the READMEs? Maybe to the canonical event model or tracker protocol docs so people have more information on the data points in the tables?

**Comment by paulboocock** [comment]
One more comment, can we move the images into a `/media` folder rather than being in the root?

File: .test/great_expectations/config/config_variables.yml (hunk 1 above)
Comment (colmsnowplow) [reply .test/great_expectations/config/config_variables.yml]: Actually this whole file shouldn't have been committed. I must have accidentally done so when I rebased. :s

File: README.md (hunk 3 above)
Comment (colmsnowplow) [reply README.md]: This one's an annoying little detail. Two things: 1. Direct link is platform specific. 2. Direct link hardcodes version (and there's no 'latest' - so I'm conscious of it getting out of date/adding to a list of things that need to be manually managed if we have a sql-runner release.

Thoughts on those?

File: README.md (hunk 2 above)
Comment (colmsnowplow) [reply README.md]: Ah my bad. Will fix.

File: README.md [hunk 4]
```diff
@@ -1 +1,36 @@
...
+
+They each also require a dataset of Snowplow events, generated by one of [the tracking SDKs](https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/), passed through the validation and enrichment steps of the pipeline, and loaded to a database.
+
+For the testing framework, Python3 is required. Install requirements with:
+
+`cd .tests`
```
Comment (paulboocock) [inline README.md]: Should be in a ```bash block as it currently displays on the same line

File: .scripts/README.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+## Setup
+
+The `run_playbooks`, `e2e` and `pr_check` scripts require SQL-runnner.
+
+They also require that the relevant template database target in the `templates/` directory is populated. One can include the password in this template, or leave it as `PASSWORD_PLACEHOLDER` and authenticate by other means - see the authentication section below.
+
+The `run_test`, `e2e` and `pr_check` scripts require `python3`. To install dependencies:
+
+`cd data-models/.test`
```
Comment (paulboocock) [inline .scripts/README.md]: Should be in a ```bash block as it currently displays on the same line

File: .scripts/README.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+bash .scripts/run_playbooks.sh ~/sql-runner redshift v1 'standard/01-base/01-base-main,standard/02-page-views/01-page-views-main,standard/03-sessions/01-sessions-main,standard/04-users/01-users-main' abcd1234;
+```
+
+## run_test.sh
+
+`run_test.sh` runs a great_expectations test suite on the output of the model. The configuration for the tests can be found in the `expectations` directory. Currently, the tests are configured to run on both temporary and permanent tables, and so should be run before the cleanup step of a model, or on a model that has `cleanup_mode` configure to `debug` or `trace`.
+
+Before running, make sure to install python requirements (python3 required):
+
+`cd data-models/.test`
```
Comment (paulboocock) [inline .scripts/README.md]: Should be in a ```bash block as it currently displays on the same line

**Review by paulboocock — COMMENTED** [review]
Nit: formatting issue

File: README.md (hunk 3 above)
Comment (paulboocock) [reply README.md]: Link here instead: https://bintray.com/snowplow/snowplow-generic/sql-runner#files ? (with or without #files in the url)

**Comment by colmsnowplow** [comment]
@alexanderdean Tagging you as I know you've taken an interest in what this repo ends up looking like. Any feedback please shout!

**Comment by alexanderdean** [comment]
Nice! I'd add the regular buttons to the top of the README and don't forget the Copyright and license section at the bottom of the README...

**Comment by alexanderdean** [comment]
Nice. I am a bit rusty but shouldn't this be flattened into a single commit against a GitHub issue?

**Comment by alexanderdean** [comment]
It would be nice to add to the README:

* A Snowplow actively maintained button, like https://github.com/snowplow/snowplow-javascript-tracker

**Comment by colmsnowplow** [comment]
> Nice. I am a bit rusty but shouldn't this be flattened into a single commit against a GitHub issue?

Yeah it will be - I just don't like to rebase when people have given feedback and might want to follow up, as it can obscure those changes. I generally do this rebase and squash last thing before the release.

> It would be nice to add to the README:

> A Snowplow actively maintained button, like https://github.com/snowplow/snowplow-javascript-tracker

Sure, I can add that - I hadn't originally as it's part of the tracker definition, but there is a shortage of applicable badges so I'll add it here too.

**Comment by colmsnowplow** [comment]
@alexanderdean so I have added that button, but on reflection I'm in two minds, for two reasons:

1. The definition of 'Actively Maintained' is production-ready. But this repo will house some alpha versions of data models, so not everything in here is 'actively maintained'. Next thing to go in here aside from web model is an alpha version of mobile model.

2. The link is to a tracker docs page, and the classifications are for trackers. Which is confusing. We could create a broader classification to apply across the estate, but feels like defining a generic one might be a bridge too far in terms of 'getting this out the door'.

Not to say I don't want to keep it. But I'm 50/50 on it. WDYT, keep it in or take it out?

**Comment by carabaestlein** [comment]
@colmsnowplow @alexanderdean I don't think the badge needs to be a blocker for the release, why don't we get this out and the mobile model as well and then decide on a general classification for the models similar to the one we have for trackers already?

**Comment by alexanderdean** [comment]
Agree with you guys, I think park it for this release and have a wider discussion later about classification or more broadly project lifecycle (WIP -> incubating -> prod -> archive?) /cc @goodits

**Comment by alexanderdean** [comment]
Copyright notice on README should be 2020 not 2012-2020

**Comment by colmsnowplow** [comment]
Done. Plan is to merge tomorrow, a little tired to be too confident in the final read through today.

**Pull request merged**